typings/index.d.ts

```diff
@@ -1216,7 +1216,7 @@ declare module 'discord.js' {
     public equals(presence: Presence): boolean;
   }
 
-  export class ReactionCollector extends Collector<Snowflake, MessageReaction> {
+  export class ReactionCollector extends Collector<Snowflake| string, MessageReaction> {
```

Comment:
I am totally not sure if this is the right move here, can someone with more knowledge verify?

Reply:
This is correct, `Collector<K, V>` defines those two methods:
https://github.com/discordjs/discord.js/blob/6a7745353207290303c1fd62967f7ee6c67c754a/typings/index.d.ts#L301-L302

Which `ReactionCollector` overrides, returning `Snowflake | string`:
https://github.com/discordjs/discord.js/blob/6a7745353207290303c1fd62967f7ee6c67c754a/typings/index.d.ts#L1232-L1233

The patch is correct, because ReactionCollector handles both snowflakes (guild emojis, aka `'773916319895781427'` for example) and emojis (twemojis/unicode, aka `'👍'`). The type here is right as the former is a `Snowflake` and the latter a `string`.